Examples/Applications/SkyboxViewer/index.js

```diff
@@ -16,8 +16,22 @@ import vtkURLExtract from 'vtk.js/Sources/Common/Core/URLExtract';
 import 'vtk.js/Sources/IO/Core/DataAccessHelper/HtmlDataAccessHelper';
 import 'vtk.js/Sources/IO/Core/DataAccessHelper/JSZipDataAccessHelper';
 
+import vtkResourceLoader from 'vtk.js/Sources/IO/Core/ResourceLoader';
+
 import style from './SkyboxViewer.module.css';
 
+// Dynamically load WebXR polyfill from CDN for WebVR and Cardboard API backwards compatibility
+if (navigator.xr === undefined) {
+  vtkResourceLoader
+    .loadScript(
+      'https://cdn.jsdelivr.net/npm/webxr-polyfill@latest/build/webxr-polyfill.js'
+    )
+    .then(() => {
+      // eslint-disable-next-line no-unused-vars, no-undef
+      const polyfill = new WebXRPolyfill();
```

Comment:
I don't think we need to bind to a name. I'm assuming that the WebXRPolyfill constructor binds to `navigator.xr`, so we can drop the `polyfill` variable and the eslint exception.

Reply:
That's fine. We still need to construct the polyfill with `new WebXRPolyfill();`, so the eslint exception `no-unused-vars` is replaced with `no-new`.